packages/playwright-core/src/client/types.ts

```diff
@@ -105,10 +105,12 @@ export type ConnectOptions = {
   timeout?: number,
   logger?: Logger,
 };
-export type LaunchServerOptions = LaunchOptions & {
+export type LaunchServerOptions = {
   host?: string,
   port?: number,
   wsPath?: string,
+  debugController?: boolean;
```

Comment:
Keep it private please.

```suggestion
  _debugController?: boolean;
```